src/tests/electron/tests/android-setup-detect-service-spinner.test.ts

```diff
@@ -10,13 +10,16 @@ import {
     simulateServiceNotInstalled,
 } from '../../miscellaneous/mock-adb/setup-mock-adb';
 
-describe('Android setup - detect-service', () => {
+const description = 'detect-service-spinner';
```

Comment:
Would it make the code more readable (especially in the calls to setupMockAdb) to give this a more descriptive name, like `testSuiteDescription` ?

Reply:
used filename as discussed offline, which removed the need for description variables